backends/blocked/ceed-blocked-operator.c

```diff
@@ -94,23 +95,24 @@ static int CeedOperatorSetupFields_Blocked(CeedQFunction qf,
     switch(eval_mode) {
     case CEED_EVAL_NONE:
       ierr = CeedQFunctionFieldGetSize(qf_fields[i], &size); CeedChkBackend(ierr);
-      ierr = CeedVectorCreate(ceed, Q*size*blk_size, &q_vecs[i]);
-      CeedChkBackend(ierr);
+      q_size = (CeedSize)Q*size*blk_size;
+      ierr = CeedVectorCreate(ceed, q_size, &q_vecs[i]); CeedChkBackend(ierr);
```

Comment:
Your choice here. It isn't necessary to make the temp variables, but they don't hurt.

Reply:
I added it because it made the casing and the purpose of the value a bit clearer to me, personally.